architectures

```diff
@@ -2,3 +2,4 @@ bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
 s390x default,onbuild,slim,stretch
+arm64 default,onbuild,slim,stretch
```

Comment:
@tianon ^ Is there some way for us to check which archs are supported for which dists?